reader.go

```diff
@@ -17,8 +17,8 @@ import (
 )
 
 const (
```

Comment:
I reversed these. Sarama and kafkacat both treat them this way (last == -1 and first == -2). Neither seem to be mentioned in the Kafka protocol, so I'm unclear whether this is a library convention or a fundamental Kafka thing.

Reply:
FWIW, I prefer 'Newest' and 'Oldest', but being in line with other libraries is probably the most important.